bigml/multivote.py

```diff
@@ -90,15 +92,75 @@ def ws_confidence(prediction, distribution, ws_z=1.96, ws_n=None):
     return (ws_p + ws_factor / 2 - ws_z * ws_sqrt) / (1 + ws_factor)
 
 
+def merge_distributions(distribution, new_distribution):
+    """Adds up a new distribution structure to a map formatted distribution
+
+    """
+    for value, instances in new_distribution.items():
+        if not value in distribution:
+            distribution[value] = 0
+        distribution[value] += instances
+    return distribution
```

Comment:
so, you're modifying the passed `distribution` and returning it, not a new one. is that intentional?